### ISSUE TITLE
Hub Reloads - Parse Load Scripts

### ISSUE THREAD

**Issue by levi-turner** [eapowertools/qs-governance-collector#35]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:

### Description

(More of a feature request / enhancement) The parse load scripts tool points to the archived path. Reloads on the Hub go to C:\ProgramData\Qlik\Sense\Log\Scripts and do not get archived. So this piece of functionality will miss out on Hub reloads. In a Development style environment a non-trivial number of apps will be reloaded frequently or exclusively on the Hub since testing a scheduled reload added no value.

### Steps to Reproduce

N/A

**Expected behavior:** N/A

**Actual behavior:** N/A

**Reproduces how often:** N/A

### Versions

N/A

### Additional Information
N/A

**Comment by goldbergjeffrey** [comment]
Good point @levi-turner .  I've noticed that script logs appear to go to multiple places.  I think the LOE on adding this directory is minimal.

**Comment by levi-turner** [comment]
Perfect. It'll get harder in multi-node since there will be Nodes+1 number of locations to parse.

GET /qrs/ServerNodeConfiguration/full will return the hostnames and a hacky way of accessing them would be to use admin shares with the parsed hostnames from the above GET call. My 2¢

**Comment by goldbergjeffrey** [comment]
what you do is put an agent on any server that would have a scripts folder, and use the web app to connect and launch the script parsing.  Then the only share required is where to deliver the parsed logs.

**Comment by rmterenzi** [comment]
This should be logged as a bug as all logs need to be rolled over in the system.

**Comment by goldbergjeffrey** [comment]
@rmterenzi logged as a bug with Qlik?  Is there a log rollover policy that is in place but may run in between running qsgc?
